lib/reform/form/validation/unique_validator.rb

```diff
@@ -1,7 +1,14 @@
 class Reform::Form::UniqueValidator < ActiveModel::EachValidator
   def validate_each(form, attribute, value)
-    if form.model.class.where("#{attribute} = ?", value).size > 0
-      form.errors.add attribute, "#{attribute} must be unique."
+    if form.model.persisted?
+      if form.model.class.where("#{attribute} = ? AND id <> ?", value, form.model.id).size > 0
```

Comment:
hey @freemanoid, I just fixed what was coded before, but what you said makes sense... I'm gonna update PR

thank you